### PR TITLE
Fixed dropdown

### LINKED ISSUE #1402: Clicking on the user profile button does not work
It is not possible to go to the user profile using the button

### PR DESCRIPTION
**Pull request by Cheshiriks**
Closed #1402

### CODE REVIEW

File: save-frontend/webpack.config.d/optimizations.js [hunk 1]
```diff
@@ -1,11 +1,3 @@
-const HtmlWebpackPlugin = require('html-webpack-plugin');
-
-config.plugins.push(
-    new HtmlWebpackPlugin({
-        template: 'index.html'
```
Comment (petertrr) [inline save-frontend/webpack.config.d/optimizations.js:5]: Problems with cache are back in 3.. 2..

Why did you need to remove it? Please describe the problem with the dropdown more specifically. `HtmlWebpackPlugin` is needed to resolve an EJS expression in `index.html` (inject current timestamp into a `<meta>` tag) to make file content refresh on every build and avoid stale cache. Which problems does it code with the existing code?

**Review by petertrr — CHANGES_REQUESTED** [review]

File: save-frontend/webpack.config.d/optimizations.js (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/webpack.config.d/optimizations.js:5]: The dropdown has the _ariaExpanded_ parameter, which changes to _true_ on click and shows a dropdown list. Now _ariaExpanded_ doesn't change on click.

Comment (petertrr) [reply save-frontend/webpack.config.d/optimizations.js:5]: Any clues about why this is happening? `HtmlWebpackPlugin` is called only during build time and in theory only affects `index.html`, where there are no references to Bootstrap or to Popper.

Comment (akuleshov7) [reply save-frontend/webpack.config.d/optimizations.js:5]: > Any clues about why this is happening? `HtmlWebpackPlugin` is called only during build time and in theory only affects `index.html`, where there are no references to Bootstrap or to Popper.

We have no idea :(

Comment (Cheshiriks) [reply save-frontend/webpack.config.d/optimizations.js]: Changed the logic for displaying Dropdown. It works now. But I still don't understand why webpack broke it.
